### PR TITLE
*: Remove owner from TODOs

### PR DESCRIPTION
**Pull request by kakkoyun**

### CODE REVIEW

**Review by v-thakkar — APPROVED** [review]
Awesome, thanks :tada:

**Review by javierhonduco — APPROVED** [review]
LGTM

File: pkg/stack/unwind/unwind_table.go [hunk 1]
```diff
@@ -155,14 +155,14 @@ func ReadFDEs(path string) (frame.FrameDescriptionEntries, error) {
 		return nil, ErrEhFrameSectionNotFound
 	}
 
-	// TODO(kakkoyun): Consider using the debug_frame section as a fallback.
-	// TODO(kakkoyun): Needs to support DWARF64 as well.
+	// TODO: Consider using the debug_frame section as a fallback.
+	// TODO: Needs to support DWARF64 as well.
```
[the remark this answers is not in the document]
Comment (kakkoyun) [reply pkg/stack/unwind/unwind_table.go:159]: I don't remember TBH. Feel free to remove them.

File: pkg/stack/unwind/unwind_table.go [hunk 2]
```diff
@@ -155,14 +155,14 @@ func ReadFDEs(path string) (frame.FrameDescriptionEntries, error) {
 		return nil, ErrEhFrameSectionNotFound
 	}
 
-	// TODO(kakkoyun): Consider using the debug_frame section as a fallback.
-	// TODO(kakkoyun): Needs to support DWARF64 as well.
+	// TODO: Consider using the debug_frame section as a fallback.
```
[the remark this answers is not in the document]
Comment (kakkoyun) [reply pkg/stack/unwind/unwind_table.go:158]: Let's remove it.

**Pull request merged**